package.json

```diff
@@ -32,21 +33,20 @@
   "bugs": {
     "url": "https://github.com/issues"
   },
-  "devDependencies": {},
   "dependencies": {
     "@babel/preset-typescript": "7.10.4",
     "@nypl/design-system-react-components": "1.2.0",
     "@nypl/dgx-react-footer": "0.5.7",
-    "@testing-library/jest-dom": "5.11.9",
-    "@testing-library/react": "11.2.3",
+    "@testing-library/jest-dom": "^6.0.0",
```

Comment:
Can you remove the carets to pin the packages? Just to not have unexpected minor changes affect the codebase later on.